### PR TITLE
prescription and dispensation using Drug

### PR DESCRIPTION
**Pull request by Guimino**
Creates Prescription and Dispensation using drug.
Created a mapping based on Regimen(concept) and DrugItem( which is one-to-one with drug) to define ARV drug Regimes.  created services and resouces to be used on POC

Removed All StockManagement Entities,

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/pharmacyapi/api/model/SupplierService.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import org.openmrs.api.db.hibernate.DbSessionFactory;
+import org.openmrs.module.pharmacyapi.api.model.stock.Supplier;
+import org.springframework.transaction.annotation.Transactional;
+
+/**
+ * @author Guimino Neves
+ */
+@Transactional
+public interface SupplierService extends OpenmrsService {
+
```
Comment (valvijo) [inline api/src/main/java/org/openmrs/module/pharmacyapi/api/model/SupplierService.java]: We don't need this class

File: api/src/main/java/org/openmrs/module/pharmacyapi/api/model/SupplierServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import org.openmrs.api.impl.BaseOpenmrsService;
+import org.openmrs.module.pharmacyapi.api.model.stock.Supplier;
+import org.springframework.transaction.annotation.Transactional;
+
+/**
+ * @author Guimino Neves
+ */
+@Transactional
+public class SupplierServiceImpl extends BaseOpenmrsService implements SupplierService {
+
```
Comment (valvijo) [inline api/src/main/java/org/openmrs/module/pharmacyapi/api/model/SupplierServiceImpl.java]: We don't need this

File: api/src/main/java/org/openmrs/module/pharmacyapi/api/model/stock/Lot.java [hunk 3]
```diff
@@ -7,96 +7,94 @@
...
 
+import org.apache.commons.lang3.builder.EqualsBuilder;
 import org.openmrs.BaseOpenmrsMetadata;
 import org.openmrs.module.pharmacyapi.api.model.DrugItem;
 
 /**
  * @author Guimino Neves
  */
 public class Lot extends BaseOpenmrsMetadata implements Serializable {
-
```
Comment (valvijo) [inline api/src/main/java/org/openmrs/module/pharmacyapi/api/model/stock/Lot.java]: We don't need this

File: api/src/main/java/org/openmrs/module/pharmacyapi/api/model/stock/StockEntry.java [hunk 4]
```diff
@@ -14,48 +15,68 @@
  * @author Guimino Neves
  */
 public class StockEntry extends BaseOpenmrsMetadata implements Serializable {
-
```
Comment (valvijo) [inline api/src/main/java/org/openmrs/module/pharmacyapi/api/model/stock/StockEntry.java]: We don't need this

File: api/src/main/java/org/openmrs/module/pharmacyapi/api/model/stock/Supplier.java [hunk 5]
```diff
@@ -14,59 +14,59 @@
  * @author Guimino Neves
  */
 public class Supplier extends BaseOpenmrsMetadata {
-
```
Comment (valvijo) [inline api/src/main/java/org/openmrs/module/pharmacyapi/api/model/stock/Supplier.java]: We don't need this

**Review by valvijo — CHANGES_REQUESTED** [review]

**Review by valvijo — APPROVED** [review]

**Comment by larslemos** [comment]
Old PR, that was fixed and Merged on PR #20

**Comment by Guimino** [comment]
this PR was moved To PR #20

**Pull request closed without merging**